src/argilla/client/feedback/training/schemas/base.py

```diff
@@ -573,22 +632,12 @@ def for_sentence_similarity(
                     _LOGGER.info(f"No label strategy defined. Using default strategy for {type(label)}.")
                 if isinstance(label, LabelQuestion):
                     label = LabelQuestionUnification(**unification_kwargs)
-                elif isinstance(label, RatingQuestion):
-                    label = RatingQuestionUnification(**unification_kwargs)
+                elif isinstance(label, RankingQuestion):
+                    label = RankingQuestionUnification(**unification_kwargs)
```

Comment:
It should be `RatingQuestion` as per https://github.com/argilla-io/argilla/pull/4171

Reply:
check